tests/fixtures/direct_award_project_list_fixture.json

```diff
@@ -1,90 +1,455 @@
+                        "id": 290439,
+                        "service": {
+                            "id": "234641366819843"
+                        }
+                    },
+                    "project": {
+                        "id": 322775348397661
+                    },
+                    "search": {
+                        "id": 3699
+                    }
+                }
+            }
+        },
+        {
+            "active": true,
+            "createdAt": "2018-06-11T11:17:13.851639Z",
+            "downloadedAt": "2018-06-11T11:17:23.967886Z",
+            "id": 379400377040928,
+            "lockedAt": "2018-06-11T11:17:19.220031Z",
+            "name": "db",
+            "outcome": {
+                "completed": true,
+                "completedAt": "2018-06-12T13:22:06.049400Z",
+                "id": 126269322275727,
+                "result": "cancelled",
+                "resultOfDirectAward": {
+                    "project": {
+                        "id": 379400377040928
+                    }
+                }
+            }
+        },
+        {
+            "active": true,
+            "createdAt": "2018-06-07T11:25:04.727593Z",
+            "downloadedAt": "2018-06-07T13:57:40.426018Z",
+            "id": 255107990746615,
+            "lockedAt": "2018-06-07T11:25:32.884392Z",
+            "name": "dfsfdsfda",
+            "outcome": null
+        },
+        {
+            "active": true,
+            "createdAt": "2018-06-05T11:28:55.824465Z",
+            "downloadedAt": "2018-06-07T14:09:20.588824Z",
+            "id": 606869699839590,
+            "lockedAt": "2018-06-07T14:09:16.255121Z",
+            "name": "fdsfs",
+            "outcome": {
+                "award": {
+                    "awardValue": "123.00",
+                    "awardingOrganisationName": "123",
+                    "endDate": "2022-12-12",
+                    "startDate": "2020-12-12"
+                },
+                "completed": true,
+                "completedAt": "2018-06-12T13:22:55.953150Z",
+                "id": 257742931727312,
+                "result": "awarded",
+                "resultOfDirectAward": {
+                    "archivedService": {
+                        "id": 273946,
+                        "service": {
+                            "id": "304481860979784"
+                        }
+                    },
+                    "project": {
+                        "id": 606869699839590
+                    },
+                    "search": {
+                        "id": 3685
+                    }
+                }
+            }
+        },
+        {
+            "active": true,
+            "createdAt": "2018-05-24T12:08:17.226976Z",
+            "downloadedAt": "2018-05-24T12:08:25.251541Z",
+            "id": 207198828251156,
+            "lockedAt": "2018-05-24T12:08:21.207534Z",
+            "name": "test",
+            "outcome": null
+        }
+    ]
+}
```

Comment:
This file is insane and needs to die 😨 

We should be using (and where necessary, building out) the api stubs in our utils repo to stop us needing files like this.

Reply:
I think we should create a tech debt ticket where we can talk about how we create test fixtures, as there seem to be a couple of different ways we use (and I didn't know about the api stubs until you mentioned it).

Reply:
Yeah I didn't know / had forgotten about the API stubs too. They also ought to move into dm-test-utils, I think?

Reply:
Oh, it uses some of the formatters defined in utils, so we'd end up with test-utils depending on utils to pull in the formatters. Not the biggest dependency, but... 🤷‍♂️

Reply:
Ah that does make sense I suppose... Ok, if I get time during firebreak I'll shift them over to test utils.